contracts/cw20-base/src/contract.rs

```diff
@@ -377,6 +380,32 @@ pub fn execute_send(
     Ok(res)
 }
 
+pub fn execute_update_minter(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    new_minter: String,
+) -> Result<Response, ContractError> {
+    let mut config = TOKEN_INFO.load(deps.storage)?;
+    // Check that sender is authorized to update minter
+    if config.mint.is_none() || config.mint.as_ref().unwrap().minter != info.sender {
```

Comment:
Instead of unwrapping, better to use `ok_or` like: https://github.com/CosmWasm/cw-plus/blob/main/contracts/cw20-base/src/contract.rs#L388-L396